src/chunkserver/raftlog/curve_segment_log_storage.cpp

```diff
@@ -422,7 +422,7 @@ int CurveSegmentLogStorage::append_entry(const braft::LogEntry* entry) {
 }
 
 int CurveSegmentLogStorage::append_entries(
-                    const std::vector<braft::LogEntry*>& entries) {
+                    const std::vector<braft::LogEntry*>& entries, braft::IOMetric* metric) {
```

Comment:
Why do you need to add a parameter? Is the interface of the new version of brake replaced

Reply:
Virtual function `virtual int append_entries(const std::vector<LogEntry*>& entries, IOMetric* metric) = 0;`, so we must break this, or do not update so high version.